package.json

```diff
@@ -97,6 +97,7 @@
     "classlist.js": "1.1.20150312",
     "cropperjs": "1.5.12",
     "custom-event-polyfill": "^1.0.7",
+    "dotenv-expand": "^5.1.0",
```

Comment:
It is supposed to be a peer dependency of this new package, we don't want to include it in the bundle itself. @eromano , do you agree?